spec/models/legislator_spec.rb

```diff
@@ -181,6 +181,24 @@
 
   end
 
+  describe "#image_url" do
+    before do
+      @legislator = FactoryGirl.build(:legislator)
+      @bioguide_id = 'b1234'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.